Apply [RFC: Universal Asset Notation] standard

Flouse opened the pull request:
- resolve https://github.com/nervosnetwork/godwoken-info/issues/39

## Reference
- https://github.com/nervosnetwork/rfcs/pull/335

## Related PR
- https://github.com/nervosnetwork/force-bridge/pull/373

Flouse committed:
Apply [RFC: Universal Asset Notation] standard

Keith-CY commented:
Updated in gwscan v1 https://v1.gwscan.com/tokens/bridge

jordanmack commented:
@Flouse A few minor issues/suggestions. 

1. IBFR starts with a lowercase "i" in the main list. UAN specifies that the asset symbol must be all capitals.
2. pCKB and TAI do not have a UAN, which is inconsistent with YOK.
3. For aesthetics, it may look better if the UAN is a separate column between Token and Address. Another option is to only include it on the details page and not on the main list at all. UAN is not really designed to be a human-friendly label so it may not belong on the main list.
4. I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge.

![20220630092308](https://user-images.githubusercontent.com/37931/176728699-3bdf2b9d-4868-4e19-ad88-076979198ec2.png)
![20220630092220](https://user-images.githubusercontent.com/37931/176728477-36d3c445-03af-49fd-9aba-46f355066a1d.png)

![20220630092657](https://user-images.githubusercontent.com/37931/176729553-5cde6338-887c-46fb-9ec2-16cb51615a1a.png)
![20220630092701](https://user-images.githubusercontent.com/37931/176729558-e7285e07-f37a-49e2-ad63-46b1b57e8de3.png)
![20220630092722](https://user-images.githubusercontent.com/37931/176729559-9007c3eb-2ef9-46d3-89b3-9136e837b388.png)

Flouse commented:
- [ ] pCKB and TAI do not have a UAN, which is inconsistent with YOK
- [ ] I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge.

Flouse commented:
> 4\. I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge. (by jordanmack)

Plan to name Godwoken Bridge as `gb` distinguishing to `gw`, so `1INCH` on GwScan will look like:
`1INCH.gw|gb.ckb|fb.eth`

> https://github.com/nervosnetwork/rfcs/pull/335#discussion_r911574244

@jordanmack what do you think?

Flouse committed:
fix(mainnet_v1): delete one unmatched token

"sudtScriptArgs": "0x160093179296127b43624ab641e2561540e18b1f1b8025d8d6830485201a16a1"

Flouse committed:
mainnet_v1: fix UAN of bridged tokens

Flouse committed:
mainnet_v1: add UAN for pCKB, dCKB and TAI

Flouse committed:
Merge branch 'main' into rfc-uan

jordanmack commented:
> Plan to name Godwoken Bridge as `gb` distinguishing to `gw`, so `1INCH` on GwScan will look like:
> `1INCH.gw|gb.ckb|fb.eth`

Yes, I think that is a good idea. I see it used as "gw" and "gb" in different places. Using "gb" is probably the intended.

Keith-CY commented:
I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

Flouse commented:
> I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

I think this slight difference won't cause confuse, so let‘s approve this PR and merge?
@Keith-CY

Keith-CY commented:
> > I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.
> 
> I think this slight difference won't cause confuse, so let‘s approve this PR and merge? @Keith-CY

That's fine

homura commented:
> I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

This is due to the `displayName` of ForceBridge using “Frocebridge” in this [commit](https://github.com/nervosnetwork/force-bridge/pull/373/files#diff-637f871f40326f9de15b973d07082ddf7f0cbcbb03357e676eb7905152cc51caR17), I think godwoken-info should be unified with FB's UAN

Keith-CY commented on mainnet_v1/bridged-token-list.json:
```diff
@@ -8,7 +8,9 @@
       "ethAddress": "0x7538C85caE4E4673253fFd2568c1F1b48A71558a"
-    }
+    },
+    "UAN": "pCKB.gw|gb.ckb",
```
Should we name it `CKB.gw|gb.ckb` since the bridged token is `CKB` but not `pCKB`

Flouse replied:
The decimals of CKB is 8, while the decimals of pCKB is 18.
